cmake/TaichiCAPI.cmake

```diff
@@ -77,6 +77,7 @@ set_target_properties(${TAICHI_C_API_NAME} PROPERTIES
     LIBRARY_OUTPUT_DIRECTORY ${C_API_OUTPUT_DIRECTORY}
     ARCHIVE_OUTPUT_DIRECTORY ${C_API_OUTPUT_DIRECTORY})
 
+if ($ENV{TI_CI})
```

Comment:
why do we want to remove this copy btw? ;)

Reply:
it basically fails raw cmake builds out of source. it's kind of a problem if you want to keep multiple cross-compiled runtime build folders on a single machine at the same time. i understand that ccache can help saving build time when we shift from a target to another but the point is i have to keep multiple build folders alive at the same time.

Reply:
hmmm then I'm a bit confused why these were added originally. cc: @bobcao3 would you mind giving some context here?

Reply:
Without this CAPI tests will fail because it can't find its DLL files. Unless we want to modify our run tests script to adapt to multi config generator.

Reply:
Also I don't get about the cross compiled part, do we support actual cross compiling？